plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -308,6 +308,9 @@
            name="%appengineNavigatorContentProvider"
            priority="high">
         <triggerPoints>
+          <reference
+                definitionId="com.google.cloud.tools.eclipse.appengine.isStandardProject">
```

Comment:
This seems to be a new addition. What for?

Reply:
suspect it's a merge conflict between me and @briandealwis 
I'll take it out.